lib/fathead/pip/parse.py

```diff
@@ -0,0 +1,138 @@
+        for n in name:
+            namestr += n + "-"
+
+        namestr = namestr[:-1]
+
+        pip_show = soup.find("div",{"id":namestr})
+
+        description = pip_show.findAll("div",{"class":"section","id":"description"})
+        usages = pip_show.findAll("div",{"class":"section","id":"usage"})
+
+        for usage in usages:
+            for child in usage:
+                try:
+                    text = child.text.strip()
+                    if(text not in hashtable):
+                        hashtable[text] = 1
+                    else:
+                        continue
+                    if(child.find('div','highlight')):
+                        fin = ''
+
+                        for t in text.split('...'):
+                            fin += t.strip() + '\n'
+
+                        fini = ''
+
+                        for t in fin.split('\n'):
+                            fini += t.strip() + '\n'
+
+                        text = '<pre><code>' + fini +'</pre></code>'
+
+                    elif(child.find('a',{"class":"toc-backref"})):
+                        text = '<h4>'+text[:-1]+'</h4>'
+
+                    for t in text.split('\n'):
+                        if(t.strip() == '' and finaltext[-4:] == '<br>'):
+                            continue;
+                        finaltext += t.strip() + '<br>'
+
+                except:
+                    pass
+
+        for section in description:
+
+            flag = 0
+            for child in section:
+                try:
+
+                    text = child.text.strip()
+                    if(text not in hashtable):
+                        hashtable[text] = 1
+                    else:
+                        continue
+                    if(child.find('div','highlight')):
+                        break;
+
+                    elif(child.find('a',{"class":"toc-backref"})):
+                        if(text[:-1] != 'Description'):
+                            break;
+                        text = '<h4>'+text[:-1]+'</h4>'
+
+                    for t in text.split('\n'):
+                        if(t.strip() == '' and finaltext[-4:] == '<br>'):
+                            continue;
+                        finaltext += t.strip() + '<br>'
+                except:
+                    pass
+        if(page == 'pip.html'):
+            finaltext += 'pip is a package management system used to install and manage software packages written in Python. Many packages can be found in the Python Package Index (PyPI). <br>'
+        
+        finaltext = '<section class="prog__container">' + finaltext + '</section>'
+
+        finaltext = ' '.join(finaltext.split())
+        finaltext = finaltext.replace('<br>','\\n')
+
+
+
+
+    usage = finaltext
+    command = ''
+
+    for p in page[:-5].split('_'):
+        command += p + ' '
+
+
+
+    command_url = baseUrl + page
+
+    list_of_data = [
+                'command ' + command,                      # unique name
```

Comment:
@karanchahal let's leave `command` out of the article titles. Instead, please create a `trigger_words.txt` file and put it in there. This will enable "command" to be at the start or end of any article title. e.g. "command pip install" or "pip install command".